app/views/miq_request/_prov_field.html.haml

```diff
@@ -258,6 +258,8 @@
                 - opts = multiple ? [] : [["<#{_('None')}>", nil]]
               - if field_hash[:data_type] == :integer
                 - opts += Array(field_hash[:values].invert).sort_by(&:last)
+              - elsif field_hash[:sort_by] == :none
```

Comment:
@lfu Wouldn't you want this to be the first check?  Otherwise and integer field could never support a `sort_by` value of `:none`.